llama/cli.py

```diff
@@ -1,6 +1,40 @@
+import os
+from datetime import datetime
+
+import boto3
 import click
 
+from llama import marc_export
+
 
 @click.group()
-def cli():
-    pass
+@click.pass_context
+def cli(ctx):
+    ctx.obj = {}
+    ctx.obj["today"] = datetime.today()
+
+
+@cli.command()
+@click.option(
+    "--export_type",
+    required=True,
+    type=click.Choice(["FULL", "UPDATE"], case_sensitive=False),
+)
+@click.option(
+    "--key_prefix",
```

Comment:
We don't need to pass the key to the command, it can be generated from the export type, eg `f"exlibris/Timdex/{export_type}/ALMA_{export_type}_EXPORT__"`.